Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source 'https://rubygems.org'
 
-ruby ENV['TRAVIS_RUBY_VERSION'] || '2.6.6'
+ruby ENV['TRAVIS_RUBY_VERSION'] || '~>2.6.6'
```

Comment:
Is there a reason we don't want to precisely pin the default Ruby version?

Reply:
I'm getting this error if it's fixed at 2.6.6.
![image](https://user-images.githubusercontent.com/31574274/113922214-e1443200-979b-11eb-82a0-daa8751925cd.png)